### PR TITLE
exiv2: fix rpath issue.

### PR DESCRIPTION
**Pull request by mahiuchun**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
Is this supposed to be a fix for ARM? Or even for existing bottles? Why not `CMAKE_INSTALL_RPATH`?

**Comment by mahiuchun** [comment]
Yes, it is a fix for ARM. Setting `CMAKE_INSTALL_RPATH` does not make it work for me.

File: Formula/exiv2.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Exiv2 < Formula
   homepage "https://www.exiv2.org/"
   url "https://www.exiv2.org/builds/exiv2-0.27.3-Source.tar.gz"
   sha256 "a79f5613812aa21755d578a297874fb59a85101e793edc64ec2c6bd994e3e778"
-  license "GPL-2.0"
+  license "GPL-2.0-or-later"
```
[the remark this answers is not in the document]
Comment (mahiuchun) [reply Formula/exiv2.rb:6]: I checked a few files in `src/` and `include/`; all top of file comments say "or later".

**Review by carlocab — APPROVED** [review]
Thanks again, @mahiuchun. Will dispatch a bottle job shortly.

**Pull request merged**